spec/models/comment_spec.rb

```diff
@@ -2,7 +2,7 @@
 
 RSpec.describe Comment, type: :model do
   user = User.create(name: 'Grabrielle', photo: 'mybaby.png', bio: 'A beautiful baby', post_counter: 0)
-  post = Post.create(user_id: author.id, title: 'Hello', text: 'This is my first post', likes_counter: 0,
+  post = Post.create(user_id: user.id, title: 'Hello', text: 'This is my first post', likes_counter: 0,
                      comments_counter: 0)
 
   comment = Comment.new(post_id: post.id, user_id: user.id, text: 'Hi Tom!')
```

Comment:
- [ ] It seems this issue was not fully resolved. There seem to be errors popping up when you run the test.  I have linked a screenshot of the errors. Kindly solve this issue. 🔽 
<img width="1579" alt="test" src="https://user-images.githubusercontent.com/32361963/229055340-cf5f4496-36c6-40f1-9daa-8ac41ee74000.png">